core/src/log.cpp

```diff
@@ -139,6 +147,40 @@ auto Logger::level() -> Logger::Level
     return state().level;
 }
 
+bool Logger::logToFile(const std::string& filePath)
+{
+    if (!FileSystem::mount("/logs", std::make_unique<cubos::core::data::StandardArchive>("./logs", /*isDirectory=*/true,
+                                                                                         /*readOnly=*/false)))
+    {
+        return false;
+    }
+
+    auto fullPath = "/logs/" + filePath;
```

Comment:
What I was saying is that we should allow users to choose any file to log to with this function.
Thus, you shouldn't mount the archive here, nor prefix the path with `/logs/`. This code should be moved to the other `logToFile` overload.